go.mod

```diff
@@ -5,7 +5,7 @@ go 1.23.7
 require (
 	github.com/confluentinc/ccloud-sdk-go-v2/apikeys v0.4.0
 	github.com/confluentinc/ccloud-sdk-go-v2/byok v0.0.2
-	github.com/confluentinc/ccloud-sdk-go-v2/certificate-authority v0.0.0-20240921001517-750d06dd7c27
+	github.com/confluentinc/ccloud-sdk-go-v2/certificate-authority v0.0.2
```

Comment:
nit: is the SDK update related to the docs update?

Reply:
nit: I was just pointing out that with an SDK update, it's no longer just a documentation update:

> Not applicable since this is docs update only.

But yes, it seems there were no significant SDK changes between v0.0.0-20240921001517-750d06dd7c27 and v0.0.2 for these two attributes. I can only quickly find [a comparison on GitHub](https://github.com/confluentinc/ccloud-sdk-go-v2/compare/certificate-authority/v0.0.1...certificate-authority/v0.0.2) 🤞.